UnityProject/Assets/Scripts/Items/Food/MeatSteak.cs

```diff
@@ -0,0 +1,25 @@
+﻿using PlayGroup;
+using UI;
+using UnityEngine.Networking;
+
+public class MeatSteak : BaseFood
+{
+    HealthBehaviour healthScript;
+    PlayerHealth playerDetails;
+    private PlayerNetworkActions playerNetworkActions;
+
+    public override void EatFood()
+    {
+
+        var currentSlot = UIManager.Hands.CurrentSlot;
+
+        currentSlot.Clear();
+
+        NetworkServer.Destroy(this.gameObject);
+        PlayerManager.LocalPlayerScript.playerNetworkActions.GetComponent<PlayerHealth>().StopBleeding();
+        PlayerManager.LocalPlayerScript.playerNetworkActions.GetComponent<PlayerHealth>().AddHealth(50);
+        PlayerManager.LocalPlayerScript.soundNetworkActions.CmdPlaySoundAtPlayerPos("EatFood");
```

Comment:
Why didn't you create  a "healing script" that could be called from all food and not just this one?

Reply:
It was not a part of this assignment? can you show me where it says I had to create a healing script?